packages/create-cloudflare/src/__tests__/deploy.test.ts

```diff
@@ -33,16 +32,24 @@ const mockInsideGitRepo = (isInside = true) => {
 describe("deploy helpers", async () => {
 	beforeEach(() => {
 		mockPackageManager("npm");
-
 		mockSpinner();
+		vi.mocked(inputPrompt).mockImplementation(async (options) => {
+			if (options.acceptDefault) {
+				return options.defaultValue;
+			}
+
+			throw new Error(
+				"If you don't want to accept the default, you must mock this function.",
+			);
+		});
```

Comment:
This emulates the behavior of `processArgument` so we can see how the `deploy` args impact the rest of the logic.